kurtosis_package/deployers/utils.star

```diff
@@ -1,13 +1,7 @@
 shared_utils = import_module("../shared_utils.star")
 
-
-# NOTE: this is a temporary workaround due to foundry-rs not having arm64 images
-FOUNDRY_IMAGE = ImageBuildSpec(
-    image_name="Layr-Labs/foundry",
-    build_context_dir="../dockerfiles/",
-    build_file="contract_deployer.Dockerfile",
-    target_stage="foundry",
-)
+# Foundry image (arm64-compatible)
+FOUNDRY_IMAGE = "ghcr.io/foundry-rs/foundry:latest"
```

Comment:
Can we use the one from `shared_utils` instead. We had it duplicated before because of issues.

Reply:
https://github.com/Layr-Labs/avs-devnet/pull/221/commits/b5ac433f2ed5e178a3d1541ad43132efafb50bb7